packages/scaffold-config/src/dependencies.ts

```diff
@@ -40,7 +40,7 @@ export const WIZARD_DEPENDENCY_REACT_DOM = {
   package: 'react-dom',
   installer: 'react-dom',
   description: 'This package serves as the entry point to the DOM and server renderers for React',
-  minVersion: '>=16.x',
+  minVersion: '>=16.0.0 || >=17.0.0 || >=18.0.0',
```

Comment:
Should this be:
```suggestion
  minVersion: '^=16.0.0 || ^=17.0.0 || ^=18.0.0',
```

Otherwise, there is no change in behavior from `>=16.x`

Reply:
Fixed!